scripts/lint-copy-mod-doc.py

```diff
@@ -25,14 +27,79 @@
             exceptions += [(ERR_LIN, fn)]
         if errno == "ERR_SAV":
             exceptions += [(ERR_SAV, fn)]
+        if errno == "ERR_RNT":
+            exceptions += [(ERR_RNT, fn)]
+        if errno == "ERR_OPT":
+            exceptions += [(ERR_OPT, fn)]
 
 new_exceptions = False
 
+def skip_comments(enumerate_lines):
+    in_comment = False
+    for line_nr, line in enumerate_lines:
+        if "/-" in line:
+            in_comment = True
+        if "-/" in line:
+            in_comment = False
+            continue
+        if line == "\n" or in_comment:
+            continue
+        yield line_nr, line
+
+def skip_string(enumerate_lines):
+    in_string = False
+    in_comment = False
+    for line_nr, line in enumerate_lines:
+        # ignore comment markers inside string literals
+        if not in_string:
+            if "/-" in line:
+                in_comment = True
+            if "-/" in line:
+                in_comment = False
+        # ignore quotes inside comments
+        if not in_comment:
+            # crude heuristic: if the number of non-escaped quote signs is odd,
+            # we're starting / ending a string literal
+            if line.count("\"") - line.count("\\\"") % 2 == 1:
+                in_string = not in_string
+            # if there are quote signs in this line,
+            # a string literal probably begins and / or ends here,
+            # so we skip this line
+            if line.count("\"") > 0:
+                continue
+            if in_string:
+                continue
+        yield line_nr, line
```

Comment:
I factored out some generator functions that attempt to skip over block comments and string literals. `skip_string` feels a bit complex, but it does reduce the number of false positives.